apps/web/src/locales/en/messages.json

```
@@ -3894,6 +3894,12 @@
       }
     }
   },
+  "encryptionKeyUpdateCannotProceed": {
+    "message": "Encryption key update cannot proceed"
+  },
+  "keyUpdateFoldersFailed": {
+    "message": "When updating your encryption key, some folders could not be decrypted. To continue with the update, these folders must be deleted. No vault items will be deleted if you proceed."
```

:art: This messaging suggests only some folders will be deleted. However we delete all folders. I believe that we need to update this messaging to reflect that?
```suggestion
    "message": "When updating your encryption key, your folders could not be decrypted. To continue with the update, all folders must be deleted. No vault items will be deleted if you proceed."
```